tests/README.md

```diff
@@ -8,6 +8,44 @@ The intial tests here verify deployment of a KFDef instance
 
 # Quick start
 
+# Running containerized tests
+
+Running the tests this way assumes that you have an active kubeadmin login
```

Comment:
So we just need to re-word to indicate peak installation is not needed for container option

Reply:
Moving the mention to the "manual" section